airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -178,14 +178,10 @@ def parse_response(
         next_page_token: Mapping[str, Any] = None,
     ) -> Iterable[Mapping]:
         for record in response.json():  # GitHub puts records in an array.
-            yield self.transform(record=record, repository=stream_slice["repository"])
-
-    def transform(self, record: MutableMapping[str, Any], repository: str = None, organization: str = None) -> MutableMapping[str, Any]:
-        if repository:
-            record["repository"] = repository
-        if organization:
-            record["organization"] = organization
+            yield self.transform(record=record, stream_slice=stream_slice)
 
+    def transform(self, record: MutableMapping[str, Any], stream_slice: Mapping[str, Any]) -> MutableMapping[str, Any]:
```

Comment:
Can we move this logic into new base class for `Repository` - dependent streams? Because this code part could be reused.